### PR TITLE
VIH-8938 - workaround to get not available endpoint

### PR DESCRIPTION
**Pull request by marcogagliardi**
[VIH-8938](https://tools.hmcts.net/jira/browse/VIH-8938)

workaround to get not available endpoint

### CODE REVIEW

File: VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/participant-waiting-room/private-consultation-participants/private-consultation-participants.component.ts [hunk 1]
```diff
@@ -150,6 +157,13 @@ export class PrivateConsultationParticipantsComponent extends WRParticipantStatu
         return availableStatuses.indexOf(participant.status) >= 0;
     }
 
+    isEndpointAvailable(endpoint: any): boolean {
+        // this is a workaround because the endpoint status when the hearing started is 'Connected'
+        const isHeartingOn = this.conference.status === ConferenceStatus.InSession;
```
Comment (oliver-scott) [inline VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/participant-waiting-room/private-consultation-participants/private-consultation-participants.component.ts:162]: typo here - `isHearingOn`

**Review by oliver-scott — COMMENTED** [review]
One minor comment, otherwise looks good

File: VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/participant-waiting-room/private-consultation-participants/private-consultation-participants.component.ts (hunk 1 above)
Comment (marcogagliardi) [reply VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/participant-waiting-room/private-consultation-participants/private-consultation-participants.component.ts:162]: ah... good spot. changed

File: VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/participant-waiting-room/private-consultation-participants/private-consultation-participants.component.ts [hunk 2]
```diff
@@ -150,6 +157,13 @@ export class PrivateConsultationParticipantsComponent extends WRParticipantStatu
         return availableStatuses.indexOf(participant.status) >= 0;
     }
 
+    isEndpointAvailable(endpoint: any): boolean {
```
[the remark this answers is not in the document]
Comment (marcogagliardi) [reply VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/participant-waiting-room/private-consultation-participants/private-consultation-participants.component.ts]: done

**Review by oliver-scott — APPROVED** [review]

**Review by komaldabhi1509 — APPROVED** [review]

**Review by shaed-parkar — APPROVED** [review]

**Pull request merged**